app/kumactl/data/install/k8s/control-plane/kuma-cp/app.yaml

```diff
@@ -17,10 +18,13 @@ spec:
     name: grpc-xds
   - port: 5679
     name: http-xds
+{{- end}}
   - port: 5681
     name: http-api-server
+{{- if ne .KumaCpMode "global"}}
   - port: 5682
     name: http-bootstrap-server
+{{- end}}
   - port: 5683
     name: gui-server
```

Comment:
So, is GUI server always exposed regardless if this is local/global/standalone?
I thought we want to expose GUI only in global/standalone.